docs/guides/high-availability/partials/network-latency.adoc

```diff
@@ -0,0 +1,7 @@
+// See https://www.cloudping.co/ for an overview of AWS latencies between AZs
```

Comment:
Isn't this actually an overview of AWS latencies between Regions? I remember making this mistake in the meeting :sweat_smile: